packages/cosmic-swingset/src/kernel-stats.js

```diff
@@ -263,36 +263,13 @@ export function exportKernelStats({
 }
 
 /**
- * Interpret the meter provider environment variables.
+ * Obtain the telemetry "meter" (really "metric") provider.
  *
  * @param {{ log: Console['log'] }} console
  * @param {Record<string, string>} env
  */
-export function getMeterProvider(console, env) {
-  if (
-    !env.OTEL_EXPORTER_PROMETHEUS_HOST &&
-    !env.OTEL_EXPORTER_PROMETHEUS_PORT
-  ) {
-    return undefined;
-  }
-  const host =
-    env.OTEL_EXPORTER_PROMETHEUS_HOST ||
-    PrometheusExporter.DEFAULT_OPTIONS.host ||
-    '0.0.0.0';
-  const port =
-    Number(env.OTEL_EXPORTER_PROMETHEUS_PORT) ||
-    PrometheusExporter.DEFAULT_OPTIONS.port;
-  const exporter = new PrometheusExporter(
-    {
-      startServer: true,
-      host,
-      port,
-    },
-    () => {
-      console.log(
-        `Prometheus scrape endpoint: http://${host}:${port}${PrometheusExporter.DEFAULT_OPTIONS.endpoint}`,
-      );
-    },
+export function getMetricProvider(console, env) {
```

Comment:
If not `meter`, I think I’d favor `metrics` over `metric`.

Reply:
Metrics it is.